### PR TITLE
Add Fake hex color functionality

### PR DESCRIPTION
**Pull request by RafaelPereiraSantos**
#17 Add Fake Hex Color
- Added a FakeColor hexadecimal value with and without the hash symbol

### CODE REVIEW

**Comment by RafaelPereiraSantos** [comment]
@CodyEngel here it its, do you have any suggestion?

**Comment by CodyEngel** [comment]
Hey @RafaelPereiraSantos the changes look good to me, only changes I could see are more stylistic ones. In the unit test for `FakeColor` I'd move the function before the tests that use it, and the mock for `FakeColor` may not require the `also`. The documentation looks good though, and I also really liked the idea of having an option to exclude the hex symbol.

I'll work on creating PR templates and setting up a style guide so it's easier for future contributors but this PR looks good to me.

**Review by CodyEngel — APPROVED** [review]
Changes look good to me @RafaelPereiraSantos -- let me know if there was anything you wanted to update or add but I'd be happy merging this in.

**Comment by RafaelPereiraSantos** [comment]
Thanks for the feedback @CodyEngel, I'm sure I can commit these changes before you merge it, since it is pretty simple I will do it right away.

It is a great idea to have a pr template and style guide! I'm looking forward to contribute with this lib in the future, I really enjoy  using the Faker in projects I work on in Ruby and I think would be great to have it available in pure Kotlin as well.

File: src/test/kotlin/dev/fakek/fakes/FakeColorTest.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+
+class FakeColorTest {
+
+    private val hexColor by lazy { "FF2D00" }
+    private val hexColorWithHashSymbol by lazy { "#$hexColor" }
+    private val fakerColor: FakerColor = mockk {
```
Comment (CodyEngel) [inline src/test/kotlin/dev/fakek/fakes/FakeColorTest.kt:13]: Nice 😎

**Comment by CodyEngel** [comment]
Awesome, the changes look good to me @RafaelPereiraSantos, I'll go ahead and merge this.

**Pull request merged**